src/pdf_manager/index.ts

```diff
@@ -303,6 +317,17 @@ export class PdfManager extends FakePdfManager implements PdfManagerContract {
       browsershot.landscape()
     }
 
+    if (!this._noSandbox) {
+      browsershot.noSandbox()
+    }
+
+    if (Object.keys(this.additionalBrowserOptions).length) {
```

Comment:
We could just use the for loop without the additional `entries` const, and conditional:

```ts
for (const [key, value] of Object.entries(this.additionalBrowserOptions)) {
  browsershot.setOption(key, value)
}
```